install/sessions.csv

```diff
@@ -190,7 +190,8 @@ HOKI,GeoNet,GNS,TRIMBLE 5700,GPS,30s,4,linz,trimble_5700 x5,2004-07-12T00:00:00Z
 HOKI,GeoNet,GNS,TRIMBLE 5700,GPS,30s,4,linz,trimble_5700 x5,2004-12-25T22:00:00Z,2007-05-14T23:00:00Z
 HOKI,GeoNet,GNS,TRIMBLE NETRS,GPS,30s,4,linz,trimble_5700 x4,2007-05-22T00:00:00Z,2011-02-08T19:43:00Z
 HOKI,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,3,linz,trimble_netr9 x4,2011-02-08T20:13:01Z,2015-08-02T17:25:53Z
-HOKI,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2015-08-02T17:25:54Z,9999-01-01T00:00:00Z
+HOKI,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2015-08-02T17:25:54Z,2017-07-17T23:48:20Z
+HOKI,GeoNet,GNS,TRIMBLE NETR9,GPS+GLO,30s,0,linz,trimble_netr9 x4,2017-07-17T23:48:21Z,9999-01-01T00:00:00Z
```

Comment:
Hi @ozym , 
HOKI had a new NetR9 deployed in 2017-07-17T23:48:21Z. Session settings hasn't changed though and I might be confused : 
did we always had a new session entry , even if the receiver model didn't change?